### DISCUSSION TITLE
Looks like a promising new UI library!

### DISCUSSION THREAD

**Discussion by cstrat** [mantinedev/mantine#6] [Feedback]
Hi team,

This looks like a very promising UI Library! Well done.
I've had a look over what it includes and it's very comprehensive already.

Questions:

   1. Is there a list of things in the works for future updates?
   2. Do you know if `<ActionIcon>` works with Font Awesome React (`@fortawesome/react-fontawesome`)?

Thanks

**Comment by cstrat** [comment]
I've had a try and using Font Awesome icons seems to work fine in `Button` & `MenuItem` elements.

**Reply by rtivital** [reply]
Hi Chris, thanks for the feedback!

You can use any react node as icon, it's not limited to any certain library, you can even use a string:

```tsx
<Button leftIcon={<FontAwesomeIcon icon="coffee" />}>Coffee</Button>
<Button leftIcon="$">Get Money!</Button>
```

The only downside of this approach – you need to set icon sizes that will match button size, example with @modulz/radix-icons:
```tsx
<Button leftIcon={<GearIcon style={{ width: 14, height: 14 }} />}>Settings</Button>
```

About future updates:
I was working in private Trello before initial release, but now I guess it's time to setup a github kanban board with project backlog, thanks for mentioning that
